pixi/src/ui/report/CoreWebVitalsReportView.js

```diff
@@ -30,7 +30,7 @@ class SimpleScale {
   render(data, unit) {
     this.bar.style.transform = `scale3d(${data.score}, 1, 1)`;
 
-    this.value.textContent = `${data.numericValue / unit.conversion}${
+    this.value.textContent = `${(data.numericValue / unit.conversion).toFixed(3)}${
```

Comment:
Suggestion: We could even do `1` or `2`. I believe PSI does `1`.

Reply:
Do we really always want to show 3 decimal places?
Instead of "3s" we would then also have "3.000s"